.vscode/settings.json

```diff
@@ -169,5 +169,4 @@
     "**/noir/noir-repo/docs/versioned_docs/**": true
   },
   "cmake.sourceDirectory": "${workspaceFolder}/barretenberg/cpp",
-  "noir.nargoPath": "./noir/noir-repo/target/release/nargo"
```

Comment:
Don't really know why this was changed at workspace level, IMHO it makes devex for us way worse. If this is something someone desires, it should be done at the user config level